CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 1.14
 Improvements:
+- Automatically configure after a configuration preset has been selected. [#2765](https://github.com/microsoft/vscode-cmake-tools/issues/2958) [@maxmitti](https://github.com/maxmitti)
```

Comment:
Automatically configure CMake project directories when the kit or the configuration preset is changed